packages/permissions/src/hooks/use-is-authorized/use-is-authorized.ts

```diff
@@ -69,6 +71,27 @@ const useIsAuthorized = ({
   selectDataFenceData?: TSelectDataFenceData;
   shouldMatchSomePermissions?: boolean;
 }) => {
+  const impliedPermissions = getImpliedPermissions(demandedPermissions);
+
+  warning(
```

Comment:
It would be nice if we can provide more "context" in the warning message, that makes it easier to identify the component where things need to be changed.

Maybe trying to get the stack trace and include it in the message

Reply:
Yes, that's why I initially wanted to use React.warn but that's not part of React anymore. I don't really see an easy way to get a hand on the component stack trace here. I would didn't check what the log looks like. Formik uses the same way to do these things if I recall correctly.